__tests__/app.test.js

```diff
@@ -217,7 +217,7 @@ describe("PATCH /api/articles/:article_id", () => {
   });
 });
 
-describe("POST /api/articles/:article_id/comments", () => {
+describe.only("POST /api/articles/:article_id/comments", () => {
```

Comment:
Get rid of this .only block and double check everything still passes before merging pls!